client/src/containers/App.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react'
 import { Route, Switch } from 'react-router-dom'
 
+<<<<<<< HEAD
```

Comment:
Looks like this merge conflict didn't get 100% resolved, so those `<` and `=` symbols will need to be removed.

If you have trouble with this, let me know and I can help fix that.